Gemfile

```diff
@@ -64,6 +64,7 @@ group :development, :test do
   gem 'pry'
   gem 'pry-byebug'
   gem 'pry-rails'
+  gem 'rubocop-rspec_rails', '2.28.2'
```

Comment:
I'm being nitpicky, but this would probably better in the `test` group above with the other `rubocop` gems. I don't think there's any major harm leaving it here though.